blueprints/going-to-production-with-healthcare-apps.html.md

```diff
@@ -7,17 +7,17 @@ redirect_from: /docs/blueprints/going-to-production-with-hipaa-apps
 
 Fly.io was built by security researchers from the ground up to be both productive and secure, making it a great home for HIPAA-compliant production healthcare applications for productive teams that ship often.
 
-This blueprint runs a developer or operations engineer through the process of evaluating Fly.io's security for HIPAA healthcare apps, launching a pilot application, signing a BAA, and deploying to production.
+This blueprint runs a developer or operations engineer through the process of evaluating Fly.io's security for HIPAA healthcare apps, launching a pilot application, signing a Business Associate Agreement (BAA), and deploying to production.
 
 ## HIPAA and Fly.io Primer
 
 The Health Insurance Portability and Accountability Act (HIPAA) sets the standard for protecting sensitive patient data. Any company that processes protected health information (PHI) must ensure that all the required physical, network, and process security measures are in place and followed.
 
-Fly.io takes a "principle of least privilege" approach to security. Here are the highlights of what healthcare app developers get out of the box on Fly.io:
+Fly.io adheres to the "principle of least privilege" approach to security. Here are the highlights of what healthcare app developers get out of the box on Fly.io:
```

Comment:
leave as is

Reply:
I have made the required change. Thank you for the review.